src/boc/BitString.ts

```diff
@@ -80,6 +80,10 @@ export class BitString {
      * @returns 
      */
     substring(offset: number, length: number) {
+        // Corner case of empty substring at the end
+        if (length === 0 && offset <= this._length) {
```

Comment:
This condition looks like useless

Reply:
I was thinking about when the bitstring has been read to the end and then the user tries to read an empty substring. Is not this a valid scenario?